integration-tests/src/test/groovy/com/okta/sdk/tests/it/FeaturesIT.groovy

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.tests.it
+
+import com.okta.sdk.resource.ResourceException
+import com.okta.sdk.resource.common.EnabledStatus
+import com.okta.sdk.resource.feature.Feature
+import com.okta.sdk.resource.feature.FeatureList
+import com.okta.sdk.tests.it.util.ITSupport
+import org.testng.Assert
+import org.testng.annotations.Test
+import wiremock.org.apache.commons.lang3.RandomStringUtils
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.*
+
+/**
+ * Tests for /api/v1/features
+ * @since 2.0.0
+ */
+class FeaturesIT extends ITSupport {
+
+    @Test
+    void featureOperationsTest() {
+        // list features
+        FeatureList featureList = client.listFeatures()
+        assertThat(featureList, iterableWithSize(greaterThanOrEqualTo(1)))
+
+        // get first feature from the list (size >= 1)
+        Feature feature = featureList.getAt(0)
+        assertThat(feature.id, notNullValue())
+
+        EnabledStatus enabledStatus = feature.getStatus()
+
+        // update a feature (lifecycle)
+        if (enabledStatus == EnabledStatus.ENABLED) {
+            // disable it
+            // "force" ensures all dependencies are also updated, if any
+            feature.updateLifecycle("false", "force")
+            assertThat(feature.getStatus(), is(EnabledStatus.DISABLED))
+        } else {
+            // enabling it programmatically may not work for all features
+            // ref: https://developer.okta.com/docs/reference/api/features/#error-response-method-not-allowed
+            try {
+                feature.updateLifecycle("true", "force")
+                assertThat(feature.getStatus(), is(EnabledStatus.ENABLED))
```

Comment:
If you go this route, pick a specific feature by ID, and toggle it.  It's likely this test would silently fail as it is

Reply:
Toggling a feature is not straightforward as it involves numerous workflow conditions as detailed at https://developer.okta.com/img/update-ssfeat-flowchart.png. I think we are better off skipping the feature lifecycle updation tests in ITs.

Reply:
+1, can you add a UT for this? (or an IT with a mocked server)

Reply:
Ack, will do.

Reply:
done

Reply:
Updated.